pmg/static/resources/css/style.scss

```diff
@@ -42,6 +42,10 @@ $yellow: #F5A623;
   font-family: $headings-font-family;
 }
 
+table {
```

Comment:
This is not necessary, we've fixed what caused this. Please make sure you're up to date from master.